configure.ac

```diff
@@ -72,6 +72,21 @@ SP_CHECK_CXXFLAGS_APPEND([OUR_CXXFLAGS], [\
                          -Wall \
                          "--param inline-unit-growth=200"])
 
+##########################################################################
+# extra API
+##########################################################################
+# usage: ./configure --enable-extra-api=yes
+SP_ARG_ENABLE_BOOL([extra_api],
+                   AC_HELP_STRING([--enable-extra-api],
+                                  [enable vma extra api support, default=no]))
+
+AS_IF([test "x$have_extra_api" = "xyes"],
+      [AC_CHECK_HEADER([mellanox/vma_extra.h],
+                       [AC_MSG_RESULT(extra api enabled)]
+                       [AC_DEFINE([USING_VMA_EXTRA_API],[1],
+                                  [using vma extra api])],
+                       [AC_MSG_ERROR([vma_extra.h file not found])])],
+      [AC_MSG_RESULT(extra api disabled)])
```

Comment:
@rafiw please consider removing https://github.com/Mellanox/sockperf/blob/sockperf_v2/configure.ac#L143 and using following construction as
AC_MSG_CHECKING(
    [if VMA Extra API is enabled])
...
    AC_MSG_RESULT([yes])
else
    AC_MSG_RESULT([no])